.ebextensions/deploy.config

```diff
@@ -0,0 +1,118 @@
+option_settings:
+  - namespace: aws:autoscaling:launchconfiguration
+    option_name: BlockDeviceMappings
+    value: /dev/sdh=:10
+  - option_name: BUNDLE_DISABLE_SHARED_GEMS
+    value: "1"
+  - option_name: BUNDLE_PATH
+    value: "vendor/bundle"
+
+packages:
+  yum:
+    git: []
+
+files:
+  "/etc/nginx/conf.d/proxy.conf" :
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      client_max_body_size  200M;
+      proxy_connect_timeout 1800;
+      proxy_send_timeout    1800;
+      proxy_read_timeout    1800;
+      send_timeout          1800;
```

Comment:
What's the difference between send_timeout and proxy_send_timeout? Does the former cover the latter or are they two distinct values?